experiments/kripke/openmp/ramble.yaml

```diff
@@ -64,7 +64,7 @@ ramble:
     packages:
       kripke:
         #pkg_spec: kripke@develop +mpi+openmp{modifier_spack_variant} ^chai@2024.02
-        pkg_spec: kripke@comm_cali +mpi+openmp{modifier_spack_variant} ^chai@2024.02
+        pkg_spec: kripke@comm_cali +caliper +mpi+openmp{modifier_spack_variant} ^chai@2024.02
```

Comment:
@gracenansamba An explicit `+caliper` variant is not needed since `{modifier_spack_variant}` provides this whenever needed

Reply:
I removed the explicit call